eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -28,11 +28,8 @@
           BeforeTargets="PrepareForILLink"> 
     <ItemGroup>
       <ManagedAssemblyToLink>
-        <TrimMode>link</TrimMode>
+        <IsTrimmable>true</IsTrimmable>
```

Comment:
Can we get rid of this whole target, and instead use `<DefaultTrimMode>link</DefaultTrimMode>`?

Reply:
Good idea (I think you mean `TrimmerDefaultAction`)

Reply:
Good suggestion. I've made the change and validated locally that the right trimaction is used for all assemblies.